hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-timelineservice-hbase/hadoop-yarn-server-timelineservice-hbase-client/src/main/java/org/apache/hadoop/yarn/server/timelineservice/storage/reader/EntityTypeReader.java

```diff
@@ -158,7 +158,9 @@ protected void validateParams() {
   private ResultScanner getResult(Configuration hbaseConf, Connection conn,
       FilterList filterList, byte[] startPrefix, byte[] endPrefix)
       throws IOException {
-    Scan scan = new Scan(startPrefix, endPrefix);
+    Scan scan = new Scan();
+    scan.withStartRow(startPrefix);
+    scan.withStopRow(endPrefix);
```

Comment:
nit: it would be the best it make it as the following:
scan.withStartRow()
          .withStopRow()
          .setFilter()
          .setSmall()